// Esercizio non svolto

// ComplexNumber
pub fn main_ex2() -> Result<String, Box<dyn std::error::Error>> {
    Ok("OK".to_string())
}